bazel/repositories.bzl

```diff
@@ -411,8 +411,12 @@ cc_library(
 def protobuf_repositories():
     native.git_repository(
         name = "protobuf_git",
-        commit = "a428e42072765993ff674fda72863c9f1aa2d268",  # v3.1.0
-        remote = "https://github.com/google/protobuf.git",
+        # Using a non-canonical repository/branch here. This is a workaround to the lack of
+        # merge on https://github.com/google/protobuf/pull/2508, which is needed for supporting
+        # arbitrary CC compiler locations from the environment. The branch is
+        # https://github.com/htuch/protobuf/tree/default-shell-env,
```

Comment:
Can you make this branch as a stable version (3.2.0) + a cherry-pick of that PR?